workers/data_refinery_workers/downloaders/sra.py

```diff
@@ -13,9 +14,25 @@
 CHUNK_SIZE = 1024 * 256
 
 
-def _download_file(file: File, downloader_job: DownloaderJob, target_file_path: str) -> bool:
+def _download_file(file: File, downloader_job: DownloaderJob, target_file_path: str, force_ftp: bool=False) -> bool:
+    """ Download file dispatcher. Dispatches to the FTP or Aspera downloader """
+
+    # SRA files have Apsera downloads.
+    if 'ftp.sra.ebi.ac.uk' in file.download_url and not force_ftp:
```

Comment:
Are there files we download that do not have this in their URL?

Reply:
Yes, I think so. At least from the tests.